app/services/chat.ts

```
@@ -152,7 +205,59 @@ export class ChatManager {
+    
+    if (!isChannelOpen) {
+      console.log('[Chat] Data channel is not in open state, current state:', this.dataChannel.readyState);
+    }
+    
+    return isChannelOpen;
+  }
+  
+  /**
+   * Wait for data channel to open (with timeout)
+   */
+  public waitForChannelReady(timeoutMs: number = 10000): Promise<boolean> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)